package.json

```diff
@@ -32,5 +32,6 @@
     "eslint-plugin-promise": "^5.1.0",
     "eslint-plugin-react": "^7.24.0",
     "typescript": "^4.3.5"
-  }
+  },
+  "dependencies": {}
```

Comment:
Is this intentional? If not, let's remove it.

Reply:
I think it was a remnant from experimenting with `node-fetch`which I had installed using `npm`. I removed it now.